### PR TITLE
Add awslogs

### PR DESCRIPTION
**Pull request by dbrock**
###### Things done

- [x] Tested using sandboxing
  ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS,
    or option `build-use-sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] Linux
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/top-level/python-packages.nix [hunk 1]
```diff
@@ -1542,6 +1542,27 @@ in {
     };
   };
 
+  awslogs = buildPythonPackage rec {
+    name = "awslogs-${version}";
+    version = "0.5";
+    src = pkgs.fetchurl {
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: fetchFromGithub is prefered here.

File: pkgs/top-level/python-packages.nix [hunk 2]
```diff
@@ -1542,6 +1542,27 @@ in {
...
+
+    doCheck = false;
+
+    propagatedBuildInputs = with self; [
+      boto3 termcolor dateutil docutils
+    ];
+
+    meta = {
+      homepage = https://github.com/jorgebastida/awslogs;
+      description = "AWS CloudWatch logs for Humans";
```
Comment (Mic92) [inline pkgs/top-level/python-packages.nix]: Do you want to maintain this package?

**Comment by Mic92** [comment]
This is not a library but a command line application, right? We put those package usually outside of python-packages.nix. Otherwise looks good to me.

**Comment by dbrock** [comment]
Ah, got it. Is this better?

**Comment by Mic92** [comment]
yeah, put it into `tools/admin`. Thanks!

**Pull request closed without merging**